client/components/club-details/EventsHostedByClub.tsx

```diff
@@ -1,6 +1,5 @@
 import styled from 'styled-components'
 import moment from 'moment-timezone'
-import { Category } from '../Category'
 import { colours, desktopFontSize, device, fontWeight, largerThan, mobile } from '../../styles'
```

Comment:
```suggestion
import { TagBubble } from '../common/TagBubble'
import { colours, desktopFontSize, device, fontWeight, largerThan, mobile } from '../../styles'
```

this is causing the Vercel deployment to fail